src/spatialdata_plot/pl/basic.py

```diff
@@ -311,53 +505,121 @@ def render_points(
+
+            elements = [elements] if isinstance(elements, str) else elements
+            if any(e not in self._sdata.images for e in elements):
+                raise ValueError(
+                    "Not all specificed elements were found, available elements are: '"
+                    + "', '".join(self._sdata.images.keys())
+                    + "'"
+                )
+
+        if channel is not None and not isinstance(channel, (list, str, int)):
+            raise TypeError("Parameter 'channel' must be a string, an integer, or a list of strings or integers.")
+
+        if isinstance(channel, list) and not all(isinstance(c, (str, int)) for c in channel):
+            raise TypeError("Each item in 'channel' list must be a string or an integer.")
+
+        if cmap is not None and not isinstance(cmap, (list, Colormap, str)):
+            raise TypeError("Parameter 'cmap' must be a string, a Colormap, or a list of these types.")
+
+        if isinstance(cmap, list) and not all(isinstance(c, (Colormap, str)) for c in cmap):
+            raise TypeError("Each item in 'cmap' list must be a string or a Colormap.")
+
+        if norm is not None and not isinstance(norm, Normalize):
+            raise TypeError("Parameter 'norm' must be of type Normalize.")
+
+        if na_color is not None and not colors.is_color_like(na_color):
+            raise ValueError("Parameter 'na_color' must be color-like.")
+
+        if palette is not None and not isinstance(palette, (list, str)):
+            raise TypeError("Parameter 'palette' must be a string or a list of strings.")
+
+        if not isinstance(alpha, (float, int)):
+            raise TypeError("Parameter 'alpha' must be numeric.")
+
+        if alpha < 0:
+            raise ValueError("Parameter 'alpha' cannot be negative.")
+
+        if quantiles_for_norm is None:
+            quantiles_for_norm = (None, None)
+        elif not isinstance(quantiles_for_norm, (list, tuple)):
+            raise TypeError("Parameter 'quantiles_for_norm' must be a list or tuple of floats, or None.")
+        elif len(quantiles_for_norm) != 2:
+            raise ValueError("Parameter 'quantiles_for_norm' must contain exactly two elements.")
+        else:
+            if not all(
+                isinstance(p, (float, int, type(None))) and (p is None or 0 <= p <= 100) for p in quantiles_for_norm
+            ):
+                raise TypeError("Each item in 'quantiles_for_norm' must be a float or int within [0, 100], or None.")
+
+            pmin, pmax = quantiles_for_norm
+            if pmin is not None and pmax is not None and pmin > pmax:
+                raise ValueError("The first number in 'quantiles_for_norm' must not be smaller than the second.")
+
+        if scale is not None and not isinstance(scale, (list, str)):
+            raise TypeError("Parameter 'scale' must be a string or a list of strings.")
+
+        if isinstance(scale, list) and not all(isinstance(s, str) for s in scale):
```

Comment:
will this break in case s is None? Bc according to the doc we allow it

Reply:
No, doesn't break. `None` toggles neither L617 or L620, so it's evaluated correctly later on 👍